.env

```diff
@@ -109,6 +109,7 @@ ESPRESSO_STATE_SIGNATURE_TOTAL_STAKE=5
 # Prover service
 ESPRESSO_PROVER_SERVICE_PORT=30050
 ESPRESSO_STATE_PROVER_UPDATE_INTERVAL=20s
+ESPRESSO_STATE_PROVER_MAX_GAS_PRICE_IN_GWEI="2.3"
```

Comment:
i set this number based on what i saw for the last month: https://etherscan.io/gastracker#chart_gasprice